src/components/PreviewOverlay.tsx

```diff
@@ -47,18 +47,6 @@ const PreviewGrid = styled.div`
   grid-template-rows: repeat(11, 1fr);
```

Comment:
The preview overlay grid structure should be same as overlay ( same number of rows and columns) otherwise the components do not look correct in the preview mode